node/cn/filters/filter_system.go

```diff
@@ -120,11 +112,10 @@ type EventSystem struct {
 //
 // The returned manager has a loop that needs to be stopped with the Stop function
 // or by stopping the given mux.
-func NewEventSystem(mux *event.TypeMux, backend Backend, lightMode bool) *EventSystem {
+func NewEventSystem(mux *event.TypeMux, backend Backend) *EventSystem {
 	m := &EventSystem{
 		mux:       mux,
 		backend:   backend,
-		lightMode: lightMode,
```

Comment:
Is lightMode related to "Remove to filter pending logs"? I'd like to know the reason for this removal

Reply:
Thanks for your attention. I guess there is no point in searching and filtering the latest logs that `lightMode` is doing with `NewHead` after removing `pending logs`. I mean, if setting `lightMode=true`, it is the same as `lightMode=false`.
(Geth [PR](https://github.com/ethereum/go-ethereum/pull/29574) doesn't mention it.)